tests/dragonfly/replication_test.py

```diff
@@ -1836,3 +1836,46 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     await c_replica.close()
     master.stop()
     replica.stop()
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
```

Comment:
If you mark this as slow it will not run in CI. Did you make sure this test does not fail with running manually the regression tests on your branch on github actions?

Reply:
i'm actually not sure what does "slow" mean. this test takes about 5 minutes, does it qualify? 

Yes, I tested manually...